### PR TITLE
Indentation correction to 2 spaces

### PR DESCRIPTION
**Pull request by shannahutch**
Let me know if this is correct.

### CODE REVIEW

File: react.html [hunk 1]
```diff
@@ -10,84 +10,84 @@
...
-  							<p>Beer and pizza are an american combo</p>
-  						</div>
-  					);
-  				}
-  			});
+    var Home = React.createClass({
+      render: function() {
+        return (
+          <div>
+  		  	<h2>Hello</h2>
```
Comment (mikemintz) [inline react.html]: You still have tabs here

**Comment by mikemintz** [comment]
The const change is good, the tabs are still a bit inconsistent

File: react.html (hunk 1 above)
Comment (shannahutch) [inline react.html]: So weird it doesn't look that way on my version.  Thank you GH?

File: react.html [hunk 2]
```diff
@@ -59,6 +59,21 @@
...
+  	  },
+  	  render: function(){
+  	  return (
+  	  	<div>
+	  	  <h1>{this.state.strikes} Lightning Strikes</h1>
+  	  	</div>
+  	  );
+  	  }
+  	});
+
```
Comment (mikemintz) [inline react.html:76]: Indentation is inconsistent

**Comment by mikemintz** [comment]
LightningCounter LGTM other than indentation

**Pull request merged**